src/main/resources/static/assets/js/sdh.js

```diff
@@ -2,22 +2,51 @@
 	new Vue({
 		el: '#contents',
 		data: {
-			fields: [ {key:'show_details', label:' ', headerTitle: 'Show Details', sortable:false}, {key:'domain', sortable:true}, {key:'date_last_assessed', sortable:true}, {key:'clinician_priority', sortable:true}, {key:'patient_readiness', sortable:true}],
+			fields: [ {key:'show_details', label:' ', headerTitle: 'Show Details', sortable:false}, {key:'domain', sortable:true}, {key:'date_last_reviewed', sortable:true}, {key:'clinician_priority', sortable:true}, {key:'patient_readiness', sortable:true}],
 			filters: {
 			      domain: '',
-			      date_last_assessed: '',
+			      date_last_reviewed: '',
 			      clinician_priority: '',
 			      patient_readiness: ''
 			    },
-	        items: [
-	            { domain: 'Food Security', date_last_assessed: '02/01/2019', clinician_priority: 'High', patient_readiness: 'Not Ready' },
-	            { domain: 'Transportation', date_last_assessed: '01/01/2019', clinician_priority: 'Low', patient_readiness: 'Motivated'},
-	            { domain: 'Housing Instability', date_last_assessed: '10/11/2018', clinician_priority: 'Medium', patient_readiness: 'Ready' },
-	            { domain: 'Utilities', date_last_assessed: '02/01/2019', clinician_priority: 'High', patient_readiness: 'Ready' },
-	            { domain: 'Interpersonal Violence', date_last_assessed: '12/01/2018', clinician_priority: 'Medium', patient_readiness: 'Motivated' },
-	            { domain: 'Education', date_last_assessed: '04/04/2018', clinician_priority: 'Medium', patient_readiness: 'Not Ready' },
-	            { domain: 'Financial Strain', date_last_assessed: '03/17/2012', clinician_priority: 'Low', patient_readiness: 'Ready' }
-	          ]
+	        domainOptions: [],
+	        priorityOptions: [],
+	        readinessOptions: [],
+	        reviews: []
+			    
+		},
+		mounted: function() {
+			const patient = this.$el.getAttribute('data-patient-id');
+			const contextPath = this.$el.getAttribute('data-context-path');
+			let _this = this;
+			$.ajax({
+				url: contextPath + '/api/sdh_domain/',
+				contentType: 'application/json',
+				success: function (data) {
+					_this.domainOptions = data._embedded.sdhDomains;
```

Comment:
Is it possible to use an arrow function for the success handler? That way you don't have to use the `_this` trick.

Reply:
Thanks. Arrow functions made it much cleaner.